app/components/TaskTemplateTile.tsx

```diff
@@ -0,0 +1,48 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { FunctionComponent } from 'react'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+
+type TaskTemplateTileTranslation = {
+  subtask: string,
+  edit: string
+}
+
+const defaultTaskTemplateTileTranslations: Record<Languages, TaskTemplateTileTranslation> = {
+  en: {
+    subtask: 'Subtasks',
+    edit: 'Edit'
+  },
+  de: {
+    subtask: 'Unteraufgabe',
+    edit: 'Bearbeiten'
+  }
+}
+
+export type TaskTemplateTileProps = {
+  templateName: string,
+  subtaskCount: number,
+  isSelected: boolean,
+  editClickedCallback: () => void,
+  tileClickedCallback: () => void
+}
+
+const TaskTemplateTile: FunctionComponent<PropsWithLanguage<TaskTemplateTileTranslation, TaskTemplateTileProps>> =
+  (props) => {
+    const translation = useTranslation(props.language, defaultTaskTemplateTileTranslations)
+    // TODO change border color for onHover and selected
+    const selectedBorderColor = props.isSelected ? 'border-black' : ''
+    return (
+      <button onClick={props.tileClickedCallback}
+              className={tw(`${selectedBorderColor} group flex flex-row rounded-md py-2 px-4 border hover:border-black justify-between items-center w-full`)}>
```

Comment:
same as in the other PR, use `tx('...', { 'border-black': isSelected)`
(also should probably not be dark but `hw-primary-700`)